pkg/syncer/admissionhandler/admissionhandler.go

```diff
@@ -249,9 +251,9 @@ func validationHandler(w http.ResponseWriter, r *http.Request) {
 			log.Debugf("admissionReview: %+v", ar)
 			switch ar.Request.Kind.Kind {
 			case "StorageClass":
-				admissionResponse = validateStorageClass(ctx, &ar)
+				admissionResponse = validateStorageClass(ctx, &ar, featureGateCsiMigrationEnabled)
```

Comment:
As these are global variables in this package, why to send them as function parameters?

Reply:
yes correct. let me not pass them as arguments.